src/r_bsp.c

```diff
@@ -302,6 +314,17 @@ static void R_MaybeInterpolateSector(sector_t* sector)
     }
 }
 
+static void R_MaybeInterpolateTextureOffsets(side_t *side)
+{
+    if (uncapped && leveltime > oldleveltime && !frozen_mode)
```

Comment:
Do we need to add more checks there? E.g. `side->basetextureoffset != side->oldtextureoffset`